bare-metal/flatcar-linux/kubernetes/variables.tf

```diff
@@ -155,6 +155,12 @@ variable "enable_aggregation" {
   default     = true
 }
 
+variable "oem_type" {
+  type        = string
+  default     = ""
+  description = "An OEM type to install with flatcar-install."
```

Comment:
Can you include the list of valid Flatcar OEMs in the description, like `(e.g. a, b, c)` and link to Flatcar Linux docs

Reply:
I wrote some more documentation here. PTAL and let me know if its okay.